common/src/main/java/com/wynntils/wc/utils/WynnItemMatchers.java

```diff
@@ -69,19 +68,19 @@ public static boolean isUnidentified(ItemStack stack) {
     }
 
     public static boolean isEmeraldPouch(ItemStack stack) {
-        return stack.getDisplayName().getString().startsWith("[§aEmerald Pouch§2 [Tier");
+        return stack.getHoverName().getString().startsWith("§aEmerald Pouch§2 [Tier");
     }
 
     public static boolean isHorse(ItemStack stack) {
         return stack.getItem() == Items.SADDLE
-                && stack.getDisplayName().getString().contains("Horse");
+                && stack.getHoverName().getString().contains("Horse");
     }
 
     /**
      * Returns true if the passed item is a Wynncraft item (armor, weapon, accessory)
      */
-    public static boolean isGear(ItemStack itemStack) {
```

Comment:
Oh, please keep the `itemStack` name. Having a `stack` is so generic, especially for what's in practice just called an "item" when you actually play Minecraft.

Reply:
In fact, I'd rather see all `ItemStack stack` changed to `ItemStack itemStack` in this file. I can do it separately, if you don't want to.

Reply:
And if `itemStack` is too long, shortening it to `item` is much better, since that is basically what it is. (The fine distinction between an `ItemStack` and an `Item` in the Minecraft code base is seldom needed, and close to never in Wynntils.)